### PR TITLE
Made menu clickable, adjusted articles

### PR DESCRIPTION
**Pull request by lindaicing**
Made menu clickable
adjusted article text
Added SEO keywords

### CODE REVIEW

**Comment by lindaicing** [comment]
**Resize interaction looks the same**
Screen Shot 2020-06-17 at 7 11 18 PM

**Article Adjustments**
Screen Shot 2020-06-17 at 7 01 12 PM
Screen Shot 2020-06-17 at 7 01 27 PM

File: Website-and-Backend/src/main/resources/public/index.html [hunk 1]
```diff
@@ -181,6 +182,9 @@ <h3>Sign up for our Mailing List!</h3>
     <div style="display:none;">
         <p>"Intro" section photo by Toa Heftiba on Unsplash</p>
         <p>"News" section photo by Lauren Fleischmann on Unsplash</p>
+        <p id="SEOKeywords">browser, chrome, consumer responsibility, corporate accountability, design, design thinking, environmental, 
+            ethical consumerism, ethically, ethical Fashion, extension, online, onlineshopping, shop, shopping, startup, sustainability, 
+            sustainable production, labor practice, practices</p>
```
Comment (CubeOfCheese) [inline Website-and-Backend/src/main/resources/public/index.html:187]: Nice! I didn't know this was a thing. We should also consider making a robots.txt file in the future

**Comment by lindaicing** [comment]
Screen Shot 2020-06-17 at 7 25 04 PM

**Comment by CubeOfCheese** [comment]
Wow these are great changes. Good work. Let's wait on this pull request until the blog post is actually posted so it can be linked.

**Comment by lindaicing** [comment]
> Wow these are great changes. Good work. Let's wait on this pull request until the blog post is actually posted so it can be linked.

Sounds good!

**Comment by lindaicing** [comment]
Forgot to screenshot (hidden) SEO keywords
Screen Shot 2020-06-17 at 7 30 41 PM

**Pull request merged**